src/firebase/shabbas-manage.ts

```diff
@@ -46,6 +46,71 @@ export const addUser = async (msg: Message) => {
   }
 };
 
+// Add alcoholic function
```

Comment:
if your code is understandable you don't need this fucking comment

Reply:
It's your code so it isn't understandable and needs the comment